src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
+  } else if (days >= 3) {
+    result = result - 20;
+  }
+
+  return result;
```

Comment:
Consider using constants for the daily cost, long term threshold, and discounts to make the code more readable and maintainable according to the checklist item 2.